modules/fedimint-mint-client/src/client_db.rs

```diff
@@ -110,3 +114,21 @@ pub async fn migrate_to_v1(
 
     Ok(None)
 }
+
+/// Maps all `Unreachable` states in the state machine to `OutputDone`
+pub(crate) fn migrate_state_to_v2(
+    operation_id: OperationId,
+    cursor: &mut Cursor<&[u8]>,
+) -> anyhow::Result<Option<(Vec<u8>, OperationId)>> {
+    let decoders = ModuleDecoderRegistry::default();
+
+    let old_state = MintOOBStatesV1::consensus_decode(cursor, &decoders)?;
```

Comment:
This is wrong, isn't it? Which type actually I need to be decoding here? @elsirion @m1sterc001guy

Reply:
I see. So it would have to be `MintClientStateMachines`. That's why, instead it is decoding the variant, so it can only deal with one type of the one variant...